CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## untagged
 
+- improve filtermail checks for encrypted messages and drop support for unencrypted MDNs
```

Comment:
Correct me, if I'm wrong, but this does affect read receipts on chatmail, right? Maybe we should make this clearer, before we're getting bug reports?
Can't really tell, if most of our users know what MDN means - I certanily didn't a few months ago.